pkg/util/proto/types.go

```diff
@@ -1,36 +1,23 @@
 package proto
 
 import (
-	"fmt"
 	"time"
 
-	"github.com/golang/protobuf/ptypes"
 	tspb "github.com/golang/protobuf/ptypes/timestamp"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 func MustTimestampProto(t time.Time) *tspb.Timestamp {
-	ts, err := ptypes.TimestampProto(t)
-	if err != nil {
-		panic(err.Error())
-	}
-	return ts
+	return timestamppb.New(t)
```

Comment:
The `ptypes/timestamp` package was updated to that `tspb.Timestamp` is an alias for `timestamppb.Timestamp. So AFAICT there's no need to keep using both packages here, we can fully switch to `timestamppb`.